*** index.html
--- index.html
***************
*** 1 ****
! 		<title>Timezone Clock</title>
--- 1 ----
! 		<title>Timezone Clock Demo</title>
***************
*** 2 ****
--- 2,11 ----
+ 		<script src="ScopedStorage.js"></script>
+ 		<script>
+			window.scopedStorage = new ScopedStorage('timezone-clock');
+ 			function resetDemo() {
+ 				scopedStorage.clear();
+ 				window.location.href = '.';
+ 			}
+ 			window.OPENWEATHERMAP_APPID = scopedStorage.getItem('appid');
+ 		</script>
  		<script src="scripts/apiproxy.php"></script>
***************
*** 2,3 ****
--- 2,6 ----
+ 		<button onclick="resetDemo()" style="position: fixed; right: .5em; bottom: 2em; z-index: 1000; padding: .5em; background-color: red;">
+ 			Reset demo
+ 		</button>
  	</body>
  </html>
